### ISSUE TITLE
Texture glitches

### ISSUE THREAD

**Issue by Anus95** [BEEmod/BEE2.4#1235]
I have been playing around with the editor and suddenly for the first time it shows very glitchy textures. Never seen before and since i downloaded the BEE2 today and this is the BEE2 test chamber, I think its an issue with BEE2

![PORTAL 2 09 01 2020 19_24_47](https://user-images.githubusercontent.com/59677445/72055090-e525a200-3315-11ea-9388-1de2ed4273ae.png)

![PORTAL 2 09 01 2020 19_24_52](https://user-images.githubusercontent.com/59677445/72055106-e9ea5600-3315-11ea-92dd-704c339d2822.png)

![PORTAL 2 09 01 2020 19_24_58](https://user-images.githubusercontent.com/59677445/72055114-ee167380-3315-11ea-881d-d4b46a90bbd8.png)

![PORTAL 2 09 01 2020 19_25_47](https://user-images.githubusercontent.com/59677445/72055119-f1a9fa80-3315-11ea-8004-e03cd6f07370.png)

![PORTAL 2 09 01 2020 19_25_55](https://user-images.githubusercontent.com/59677445/72055124-f4a4eb00-3315-11ea-9243-0f2b37a42d8f.png)

**Comment by Flash4433** [comment]
Those weird textures are called "leaks". Might be because you're using overgrown, it usually causes issues. Try the Clean style.

**Comment by Anus95** [comment]
Thank you, though is there a way to get rid of them while still using overgrown?

**Comment by LautaroL20** [comment]
Were you using the "Half-Grate" item in your map?

**Comment by Anus95** [comment]
No, all i have used is: Funnel, buttons, pedestal buttons,orange auto portal, repulsion gel, glass panel, goo and the cube, I also had the glass and the glass hole before in the level but i deleted it.

**Comment by Flash4433** [comment]
At this point in time, I don't think you can easily fix these issues. I recommend using another style, however, if you really want to keep that style, try changing the entrance/exit, maybe removing the goo too, although that would definitely change the puzzle...

But yeah, I really recommend you use another style. I also really like Overgrown, but leaks don't allow a map to publish to workshop. By the way, what Skybox have you selected?

**Comment by LautaroL20** [comment]
I will transfer your issue to the correct repository, since you are posting in the application section. Anything related to items & styles must be posted on the item´s page, here: https://github.com/BEEmod/BEE2-items/issues

This section is for post things related to the Bee´s user interface, only.

**Comment by Anus95** [comment]
the overgrown one as well
